apps/hash-frontend/src/shared/icons/calendar-day-light-icon.tsx

```diff
@@ -0,0 +1,16 @@
+import { SvgIcon, SvgIconProps } from "@mui/material";
+import { FunctionComponent } from "react";
+
+export const CalendarDayLightIcon: FunctionComponent<SvgIconProps> = (
+  props,
+) => {
+  return (
+    <SvgIcon {...props} height="16" width="14" viewBox="0 0 448 512">
+      <path
+        opacity="1"
```

Comment:
Is this required on these icons?

Reply:
I don't think so, FA now attaches this for some reason to the SVGs when downloading. Have removed them in https://github.com/hashintel/hash/pull/3685/commits/953519aba56392a8ab28e29048980fa88023166c